iotdb/src/main/java/org/apache/iotdb/db/metadata/Metadata.java

```diff
@@ -67,6 +70,57 @@ public Metadata(Map<String, List<MeasurementSchema>> seriesMap,
     return deviceIdMap;
   }
 
+  /**
+   * combine multiple metadatas
+   */
+  public static Metadata combineMetadatas(Metadata[] metadatas) {
```

Comment:
This is a public method, you should add more unit tests

Reply:
Besides, i am wondering why you provide a combine function rather than a add function
```java
public void addMetadata(Metadata m){
}
```

Reply:
Added UT for this.
Because it is only used if multiple Metadatas need to be combined, a add function is not suitable for this.